src/components/cards/experience-listing-card/ExperienceListingCard.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div class="experience-listing-card">
+    <div class="container">
+      <div class="image">
+        <img :src="image" />
+      </div>
+
+      <div class="body">
+        <p
+          :class="['status', isListed ? 'isListed' : '']"
+          v-if="deviceSize <= deviceSizes.s"
+        >{{ checkListed }}</p>
+        <p class="title">{{ title }}</p>
+        <p class="country" v-if="deviceSize > deviceSizes.s">{{ country }}</p>
+        <div class="actions" v-if="deviceSize <= deviceSizes.s">
+          <div>
+            <a>{{ listingAction }}</a>
+          </div>
+          <template>
+            &nbsp;&bull;&nbsp;
+            <div>
+              <a>View</a>
+            </div>
+          </template>
+        </div>
+      </div>
+
+      <div class="wrapper" v-if="deviceSize > deviceSizes.s">
+        <div class="actions">
+          <div>
+            <a>edit</a>
+          </div>
+          <template v-if="isListed">
+            &nbsp;&bull;&nbsp;
+            <div>
+              <a>preview</a>
+            </div>
+          </template>
+        </div>
+        <div :class="['status', isListed ? 'isListed' : '']">
+          <p>{{ checkListed }}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { responsiveHandler } from "../../../mixins/responsiveHandler";
+
+export default {
+  name: "experience-listing-card",
+  mixins: [responsiveHandler],
+  data() {
+    return {
+      status: ""
+    };
+  },
+  props: {
+    image: { type: String },
+    isListed: { type: Boolean },
+    title: { type: String },
+    country: { type: String }
+  },
+  computed: {
+    checkListed() {
+      return this.isListed ? (status = "Listed") : (status = "Unlisted");
+    },
+    listingAction() {
+      return this.isListed ? "Unlist" : "List";
+    }
+  },
+  methods: {
+    openExperiencePage() {
```

Comment:
This name mean that you will open a page (which is a business logic btw) or her you just emit a event when someone is clicking